cruise-control/src/yaml/endpoints/load.yaml

```diff
@@ -0,0 +1,64 @@
+LoadEndpoint:
+  get:
+    operationId: load
+    summary: Get the cluster load once Cruise Control Load Monitor is running.
+    parameters:
+      - name: start
+        in: query
+        description: Start time of the cluster load.
+        schema:
+          type: long
+          format: int64
```

Comment:
can we add a `minimum` and `default` field for this parameter? (see [partitionLoad.yaml](https://github.com/linkedin/cruise-control/blob/b15fbffd04b8bd4fa561e13a3bc247f02fc794dc/cruise-control/src/yaml/endpoints/partitionLoad.yaml))